pipeline/luigi/CompileVCFFiles.py

```diff
@@ -1447,6 +1447,60 @@ def run(self):
 
 
 @requires(RunDiffAndAppendChangeTypesToOutput)
+class RunDiffAndAppendChangeTypesToOutputReports(luigi.Task):
+    def _extract_release_date(self, version_json):
+        with open(version_json, 'r') as f:
+            j = json.load(f)
+            return datetime.datetime.strptime(j['date'], '%Y-%m-%d')
+
+
+    def _extract_file(self, archive_path, tmp_dir, file_path):
+        with tarfile.open(archive_path, "r:gz") as tar:
+            tar.extract(file_path, tmp_dir)
+
+        return tmp_dir + '/' + file_path
+
+
+    def output(self):
+        release_dir = self.output_dir + "/release/"
+        diff_dir = create_path_if_nonexistent(release_dir + "diff/")
+        return {'reports_with_change_types': luigi.LocalTarget(release_dir + "reports_with_change_types.tsv"),
+                'removed_reports': luigi.LocalTarget(diff_dir + "removed_reports.tsv"),
+                'added_reports': luigi.LocalTarget(diff_dir + "added_reports.tsv"),
+                'added_data_reports': luigi.LocalTarget(diff_dir + "added_data_reports.tsv"),
+                'diff_reports': luigi.LocalTarget(diff_dir + "diff_reports.txt"),
+                'diff_json_reports': luigi.LocalTarget(diff_dir + "diff_reports.json"),
+                'README': luigi.LocalTarget(diff_dir + "README.txt")}
+
+    def run(self):
+        release_dir = self.output_dir + "/release/"
```

Comment:
maybe for a next time: I think it's possible to work with these Target objects returned by `self.output()`. In case of a single return, one can just do `self.output().path` to refer to the path. In this case, would need to lookup the target objects in the dictionary. This way, path logistics are not duplicated  
According to http://luigi.readthedocs.io/en/stable/api/luigi.local_target.html directories could also be created, if necessary.

Reply:
sounds like a nice cleanup option for future development :) I'm going to omit this for now